RecoParticleFlow/PFClusterProducerAlpaka/interface/alpaka/PFClusterProducerKernel.h

```diff
@@ -0,0 +1,93 @@
+#ifndef RecoParticleFlow_PFClusterProducerAlpaka_PFClusterProducerAlpakaKernel_h
+#define RecoParticleFlow_PFClusterProducerAlpaka_PFClusterProducerAlpakaKernel_h
+
+#include "DataFormats/ParticleFlowReco/interface/alpaka/PFRecHitDeviceCollection.h"
+#include "DataFormats/ParticleFlowReco/interface/PFRecHitHostCollection.h"
+#include "DataFormats/ParticleFlowReco/interface/alpaka/PFClusterDeviceCollection.h"
+#include "DataFormats/ParticleFlowReco/interface/alpaka/PFRecHitFractionDeviceCollection.h"
+#include "RecoParticleFlow/PFClusterProducerAlpaka/interface/alpaka/PFClusterParamsAlpakaESData.h"
+#include "RecoParticleFlow/PFClusterProducerAlpaka/interface/alpaka/tmpDeviceCollection.h"
+#include "RecoParticleFlow/PFClusterProducerAlpaka/interface/alpaka/tmpEdgeDeviceCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  typedef struct float4 {
+    float x;
+    float y;
+    float z;
+    float w;
+  } float4;
```

Comment:
How about
```suggestion
 struct Float4 {
    float x;
    float y;
    float z;
    float w;
  };
```
?

The type names seem pretty generic, I think it would be better to move them to more specific namespace.